includes/class-wcpdf-settings-callbacks.php

```diff
@@ -412,6 +428,15 @@ public function media_upload( $args ) {
 	 */
 	public function next_number_edit( $args ) {
 		extract( $args ); // $store, $size, $description
+
+		if ( ! empty( $store_callback ) ) {
+			if( $store_callback instanceof Sequential_Number_Store ) {
```

Comment:
`$store_callback` should never be a class instance, _unless_ that class is callable (meaning it has an `__invoke()` method). It's either `$store` or `$store_callback`, the second should always be a callback!